vars/signArtifacts.groovy

```diff
@@ -0,0 +1,25 @@
+void call(Map args = [:]) {
```

Comment:
Lets moving this change into its own PR so it doesn't impact the python workflow that is ready to merge